pkg/snapshot/clonetree_pci_linux.go

```diff
@@ -70,6 +70,19 @@ func scanPCIDeviceRoot(root string) (fileSpecs []string, pciRoots []string) {
 		"revision",
 		"vendor",
 	}
+
+	perDevEntriesOpt := []string{
+		"driver",
+		"net/*",
+		"physfn",
+		"sriov_*",
+		"virtfn*",
+	}
+
+	ignoreSet := newKeySet(
```

Comment:
I don't think it's necessary to create a new `keySet` typedef. We can just use `map[string]bool` here and where you're doing the `ignoreSet.Contains` call below, just do:

```go
if _, ignored := ignoreSet[globbedEntry]; ignored {
     continue
}
```

Reply:
sure thing, will simplify